eventlet/timeout.py

```diff
@@ -175,5 +174,4 @@ def fun(*args, **kwargs):
 
 
 def is_timeout(obj):
-    py3err = getattr(__builtins__, 'TimeoutError', Timeout)
-    return bool(getattr(obj, 'is_timeout', False)) or isinstance(obj, py3err)
+    return bool(getattr(obj, 'is_timeout', False)) or isinstance(obj, getattr(__builtins__, 'TimeoutError', Timeout))
```

Comment:
This needs to be supported by numbers.

Reply:
this one is pretty simple, to agree upon.
lighter condition should come first on OR operator
the py3err  happens to be getattr+isinstance

Reply:
This is how support by numbers works:
- you take a real-like program and make it as determinant as possible: remove network, disk, human operations
- run program few times and measure CPU time, numbers from different runs must be close, otherwise we don't have a stable base to compare against
- change code
- run program few times and measure CPU time
- compare CPU times before and after change

Reply:
It is OK, to decide over two different methods and so on,
while, if the same operation is faster against additional operation to the same, there is nothing to check about.